bot/__main__.py

```diff
@@ -0,0 +1,21 @@
+"""This is the main module of the SAM project."""
+
+from discord.ext import commands
+import settings
+
+bot = commands.Bot(command_prefix=settings.BOT_PREFIX)
+initial_extensions = ['bot.util.utilities']
+
+@bot.event
+async def on_ready():
+    """Event handler for Bot entering ready state
+    """
+    print('- {0.user} successfully logged in to Discord!'.format(bot))
```

Comment:
what does {0.user} do?

Reply:
https://www.w3schools.com/python/ref_string_format.asp
In general the same as `bot.user`.